src/app/explore/page.tsx

```diff
@@ -55,7 +55,7 @@ export default function Page() {
     console.log(res)
     userStore.init(res.data.created_user)
 
-    push('/dashboard')
+    push('/welcome')
```

Comment:
Add a comment: 

//TODO: /explore/welcome if user is signup 
//TODO: /dashboard if user signin

Reply:
@utkarsh-1602 kindly make the changes. Also refactor welcome inside explore folder.

it should be /explore/welcome.